tests/cpp/testobject/PdxVersioned1.cpp

```diff
@@ -131,62 +131,62 @@ void PdxVersioned1::init(const char* key) {
   m_float = 23324.324f;
   m_double = 3243298498.00;
   m_string = std::string("PdxVersioned ") + key;
-  m_boolArray = new bool[3];
+  m_boolArray = std::vector<bool>(3);
   m_boolArray[0] = true;
   m_boolArray[1] = false;
   m_boolArray[2] = true;
   /*for(int i=0; i<3; i++){
     m_boolArray[i] = true;
   };*/
 
-  m_byteArray = new int8_t[2];
+  m_byteArray = std::vector<int8_t>(2);
   m_byteArray[0] = 0x34;
   m_byteArray[1] = 0x64;
 
-  m_sbyteArray = new int8_t[2];
+  m_sbyteArray = std::vector<int8_t>(2);
```

Comment:
try:
```
m_sbyteArray = std::vector<int8_t>{0x34, 0x64};
```

Reply:
Ah, thanks.  That's what I was looking for!